Source/WebCore/page/ChromeClient.h

```diff
@@ -683,9 +683,11 @@ class ChromeClient {
 
     virtual void addInitialTextAnimationForActiveWritingToolsSession() { }
 
-    virtual void addSourceTextAnimationForActiveWritingToolsSession(const CharacterRange&, const String&, CompletionHandler<void(TextAnimationRunMode)>&&) { }
+    virtual void addSourceTextAnimationForActiveWritingToolsSession(const WTF::UUID& /*sourceAnimationUUID*/, const WTF::UUID& /*destinationAnimationUUID*/, bool, const CharacterRange&, const String&, CompletionHandler<void(TextAnimationRunMode)>&&) { }
```

Comment:
You can remove the `/* */` and simply name the variable.

Reply:
Other variables with names in this file are done with comments, so sticking with that.

Reply:
(it should be `/* destinationAnimationUUID */` instead of `/*destinationAnimationUUID*/`